radis/lbl/base.py

```diff
@@ -1873,7 +2014,11 @@ def _calc_Q(molecule, iso, state):
         # Einstein A coefficient and the populations (see Klarenaar 2017 Eqn. 12)
 
         # correct for Partition Function
-        line_strength = df1.int * (df1.Qref / df1.Qgas)
+        if "iso" in df1:
```

Comment:
for all these "if 'iso' in df1" lines , i think a comment "#multiple-isotopes in database" would be nice for future develoeprs !